README.md

```diff
@@ -2,3 +2,30 @@ FuseSoC standard core library
 =============================
 
 This is the standard core library to be used with [FuseSoC](https://github.com/olofk/fusesoc)
+
+### Installation
+
+Clone this repository and add it to your `fusesoc.conf`.  For example
```

Comment:
This library will be automatically cloned to `~/.local/share/fusesoc/fusesoc-cores` and added to fusesoc.conf when you run `fusesoc init`. We can keep the current instructions as an alternative/manual installation